policyengine_us/parameters/gov/states/ma/tax/income/credits/dependent/amount.yaml

```
@@ -1,12 +1,19 @@
 description: MA dependent credit amount.
 values:
   2021-01-01: 180
+  2023-01-01: 310
 metadata:
   reference:
     - title: Form 1 2021 Massachusetts Resident Income Tax (line 46)
       href: https://www.mass.gov/doc/2021-form-1-instructions/download#page=17
     - title: Form 1 2022 Massachusetts Resident Income Tax (line 46)
       href: https://www.mass.gov/doc/2022-form-1-instructions/download#page=17
+    - title: Draft Form 1 2023 Massachusetts Resident Income Tax (line 46)
+      href: https://www.mass.gov/doc/draft-2023-form-1-instructions/download#page=17
+    - title: Law	Result Summary Part I Title IX Chapter 62 Section 6 (y) (ii) (B)
```

Please do the same for the cap. And doesn't the bill change the age limit from under 12 to under 13? In any case, please add the legal reference to the age limit while we're here.
```suggestion
    # Does not yet reflect 2023 changes as of 2023-12-14.
    - title: Massachusetts General Laws Part I Title IX Chapter 62 Section 6 (y) (ii) (B)
```